<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ulimitCheck</class>
 <widget class="QWidget" name="ulimitCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>240</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文中宋</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>系统内核参数检查</string>
  </property>
  <widget class="QGroupBox" name="groupBoxOutside">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>577</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: lightgray;</string>
   </property>
   <property name="title">
    <string>系统内核参数检查工具</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="labelCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
font: 75 11pt &quot;微软雅黑&quot;;
background-color: white;</string>
    </property>
    <property name="text">
     <string>选项检查</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
font: 75 11pt &quot;微软雅黑&quot;;
background-color: gold;
selection-background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>开始检查</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCance">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
font: 75 11pt &quot;微软雅黑&quot;;
background-color: gold;</string>
    </property>
    <property name="text">
     <string>点击取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxInside">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: white;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBoxUlimit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ulimit参数</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
